app/src/main/java/org/mozilla/focus/fragment/ScreenCaptureDialogFragment.java

```diff
@@ -57,6 +58,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.dialog_capture_screen, container, false);
         mLottieAnimationView = (LottieAnimationView) view.findViewById(R.id.animation_view);
+        if (!Inject.isScreenCaptureAnimationEnabled()) {
```

Comment:
can you inject the playback condition around playAnimation() instead of setRepeatCount()? So the original code flow will be similar. 
I also suggest renaming the method isScreenCaptureAnimationEnabled to isUnderEspressoTest for better understanding why we want to disable the animation in EspressoTest.